lib/systems/default.nix

```diff
@@ -189,6 +189,9 @@ rec {
         then "${pkgs.mmixware}/bin/mmix"
         else throw "Don't know how to run ${final.config} executables.";
 
+        emulatorAvailable = pkgs:
```

Comment:
I'd rather replace the `throw` with `null` – that will also fail any string interpolation (which is required for emulator always anyways).

`tryEval` is arguably a bit of a code smell and it will also not always work, e.g. often `targetPackages` will be basically empty which we also can't catch (but could in the emulator function).

Reply:
how about now?

i want to keep the throw to have a clear message of why it fails instead of just `error: cannot coerce null to a string`